docs/rackhd/event_notification.rst

```diff
@@ -244,33 +244,43 @@ The `payload.json` attributes in the example above are as below:
 ========= ====== ============ ============================================
 Attribute Type   Flags        Description
 ========= ====== ============ ============================================
-url       String **required** The hook url that events are notified to.
-name      String **optional** Any name user specified.
-filters   Array  **optional** The conditions of which events are notified. When it's not specified, or it's empty, all events will be notified to registered `url`
+url       String **required** The hook url that events are notified to. Both http and https urls are supported.
```

Comment:
Add some words to say the `url` must be unique across different hooks, this is to avoid send duplicated events to the same url.